amr-wind/wind_energy/actuator/turbine/fast/FastIface.H

```diff
@@ -67,13 +67,13 @@ protected:
 
     // Not using amrex::Real. Instead use explicit types from FAST API
     double m_start_time{0.0};
-    double m_stop_time;
+    double m_stop_time{-1.0};
     SimMode m_sim_mode{SimMode::init};
 
     int m_num_sc_inputs{0};
     int m_num_sc_outputs{0};
 
-    char m_errmsg[fast_strlen()];
+    // char m_errmsg[fast_strlen()];
```

Comment:
This line can be removed entirely. Superseded by https://github.com/Exawind/amr-wind/blob/dc887e210c9ad99e2fd7225a6d91b9826c350d72/amr-wind/wind_energy/actuator/turbine/fast/FastIface.cpp#L18

Reply:
Thanks. Wasn't sure why it was there.